Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/Home.kt

```diff
@@ -198,72 +211,106 @@ fun HomeContent(
+                        item {
+                            FollowedPodcastItem(
+                                items = featuredPodcasts,
+                                pagerState = pagerState,
+                                onPodcastUnfollowed = onPodcastUnfollowed,
+                                modifier = Modifier
+                                    .fillMaxWidth()
+                                    .verticalGradientScrim(
+                                        color = scrimColor,
+                                        startYPercentage = 1f,
+                                        endYPercentage = 0f
+                                    )
+                            )
+                        }
 
-                HomeAppBar(
-                    backgroundColor = appBarColor,
-                    modifier = Modifier.fillMaxWidth()
-                )
+                        if (isRefreshing) {
+                            // TODO show a progress indicator or similar
+                        }
 
-                if (featuredPodcasts.isNotEmpty()) {
-                    Spacer(Modifier.height(16.dp))
+                        stickyHeader {
+                            if (homeCategories.isNotEmpty()) {
+                                HomeCategoryTabs(
+                                    categories = homeCategories,
+                                    selectedCategory = selectedHomeCategory,
+                                    onCategorySelected = onHomeCategorySelected
+                                )
+                            }
+                        }
 
-                    FollowedPodcasts(
-                        items = featuredPodcasts,
-                        pagerState = pagerState,
-                        onPodcastUnfollowed = onPodcastUnfollowed,
-                        modifier = Modifier
-                            .padding(start = Keyline1, top = 16.dp, end = Keyline1)
-                            .fillMaxWidth()
-                            .height(200.dp)
-                    )
+                        when (selectedHomeCategory) {
+                            HomeCategory.Library -> {
+                                // TODO
+                            }
 
-                    Spacer(Modifier.height(16.dp))
+                            HomeCategory.Discover -> {
+                                discoverItems(
+                                    discoverViewState = discoverViewState,
+                                    podcastCategoryViewState = podcastCategoryViewState,
+                                    navigateToPlayer = navigateToPlayer,
+                                    onCategorySelected = onCategorySelected,
+                                    onTogglePodcastFollowed = onTogglePodcastFollowed
+                                )
+                            }
+                        }
+                    }
                 }
             }
         }
+    }
+}
```

Comment:
nit: This is super nested, consider breaking it apart into multiple composables where it makes sense